src/connectors/current-refined-values/connectCurrentRefinedValues.js

```diff
@@ -214,3 +218,14 @@ function clearRefinementFromState(state, refinement) {
 function clearRefinement(helper, refinement) {
   helper.setState(clearRefinementFromState(helper.state, refinement)).search();
 }
+
+function computeLabel(value) {
```

Comment:
Do you mean to externalize somewhere else this logic or a way to override this function through connector options?